Workstream Policy.md

```diff
@@ -318,6 +318,21 @@ case-by-case basis by the [Steering Group].
 [Other Publications] published by the WHATWG bear text to be determined by the
 [Steering Group].
 
+### Notice for Translations
+
+WHATWG [Living Standards][Living Standard] are under a permissive license that allows
+independent translations. WHATWG recommends the following notice for translations, including the link:
+
+> This is a translation of X, which is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Copyright © YEAR WHATWG (Apple, Google, Mozilla, Microsoft).
```

Comment:
I think per https://wiki.creativecommons.org/wiki/Best_practices_for_attribution the last sentence (and its attendant fun update-every-year implications) is not necessary. Maybe instead "This is a translation of the WHATWG X Standard, which is licensed under a ..."

Reply:
Hmm. I originally got the text I put in from @annevk, who said it was CC's own recommendation for translations, but I am not sure where that recommendation ultimately comes from.

Reply:
I was just trying to infer what they recommended. I wasn't sure myself (hence the question mark). Dropping the copyright line seems fine if that's not needed.

Reply:
I suspect it may make the lawyers twitchy if we encourage people to remove the full copyright notice from derived works. Of course, legally, they only have to do what the license requires. But since this is a recommendation and not a hard requirement, it seems good to recommend the best practice.

Reply:
If I'm reading this correctly, @othermaciej's approach makes sense both in terms of maintaining the attribution and having translations refer to the original as the source material; if a translation introduces an error, the recipient of the translation at least knows where to find the original.